### DISCUSSION TITLE
is server changes ip

### DISCUSSION THREAD

**Discussion by emerymastering** [apohl79/audiogridder#1078] [Help]
I share internet with server computer sometimes and then it get a different ip. Once i disable the sharing of internet it gets local ip.
But after that the project doesn't connect automatically and I need to do it manually. Is it possible to make that it finds server automatically even after IP change?

**Comment by rncwalker** [comment]
Change the Server IP to a Static IP (Never changes).

Robert :)

**Comment by apohl79** [comment]
what version are you on? and what version did you save the session with? since 1.2-b10, a server generates a unique id for itself, so a plugin can reconnect even if the ip changes.
